### PR TITLE
Support central conf package for ETL app.

### PR DESCRIPTION
**Pull request by pwalsh**
Relates to https://github.com/whiletrue-industries/srm-etl/pull/4

Open questions:

1. The Helm config for the ETL service currently refers to both `GOOGLE_KEY` and `GOOGLE_API_KEY`. The Docker Compose section of README only refers to `GOOGLE_KEY` and `GOOGLE_SECRET`. I'm not sure if `GOOGLE_KEY` and `GOOGLE_API_KEY` are the same thing, or different.
2. I don't know where environment variables are actually set for deployment. The docs refer to an optional `.env` file that is kept out of versioning, but I assume that is for deploying from a developer machine and not CI.

### CODE REVIEW

**Comment by akariv** [comment]
`GOOGLE_API_KEY` is the require env var that is used by the `geocoder` package when using the Google Maps provider (https://geocoder.readthedocs.io/providers/Google.html#environment-variables).
However, we can probably change that to something more meaningful and instantiate the geocoder with the `key` argument, passing the api key explicitly.

**Comment by akariv** [comment]
All actual secrets are managed in kubernetes directly - I'll explain in private.

File: .etl.env [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+ETL_MAPBOX_ACCESS_TOKEN=
+ETL_MAPBOX_LIST_TILESETS=https://api.mapbox.com/tilesets/v1/srm-kolzchut
+ETL_MAPBOX_UPLOAD_CREDENTIALS=https://api.mapbox.com/uploads/v1/srm-kolzchut/credentials
+ETL_MAPBOX_CREATE_UPLOAD=https://api.mapbox.com/uploads/v1/srm-kolzchut
+ETL_MAPBOX_UPLOAD_STATUS=https://api.mapbox.com/uploads/v1/srm-kolzchut/
+
+ETL_DATAFLOWS_AIRTABLE_APIKEY=
+
+GOOGLE_API_KEY=
```
Comment (akariv) [inline .etl.env:27]: See comment on other PR - we should probably rename this to `ETL_GOOGLE_MAPS_API_KEY` (while fixing the code accordingly) for consistency

**Comment by akariv** [comment]
Before merging this, let's make sure the staging secrets are in line with these changes.

**Comment by pwalsh** [comment]
@akariv updated.

**Comment by akariv** [comment]
yes, some env vars are part of the DGP platform itself (e.g. GOOGLE_KEY and GOOGLE_SECRET relate to user authentication), as described here: https://github.com/whiletrue-industries/srm-etl#environment-variables

I've asked @OriHoch to provide you with credentials for accessing the project on kubernetes (via rancher), so you'll be able to make these changes yourself.

**Comment by pwalsh** [comment]
@akariv updated on k8s. Left all existing env vars and will clean them up after a future deployment.

**Pull request merged**